### PR TITLE
Add AntiForgery tokens to some actions that were missing them

### PR DESCRIPTION
**Pull request by scottbommarito**
Mostly admin routes.

These include
- LuceneController
- SupportRequestController
- SecurityPolicyController
- CuratedPackagesController
- UsersController

https://github.com/NuGet/Engineering/issues/638

### CODE REVIEW

**Comment by skofman1** [comment]
How did you test this?

File: src/NuGetGallery/Areas/Admin/Views/SecurityPolicy/Index.cshtml [hunk 1]
```diff
@@ -68,7 +72,13 @@
 @section BottomScripts {
     <script src="@Url.Content("~/Scripts/knockout-2.2.1.js")"></script>
     <script>
-        $(document).ready(function() {
+        $(document).ready(function () {
+            var addAntiForgeryToken = function (data) {
+                var $field = $("#AntiForgeryForm input[name=__RequestVerificationToken]");
```
Comment (joelverhagen) [inline src/NuGetGallery/Areas/Admin/Views/SecurityPolicy/Index.cshtml]: This code should go in a shared location.

**Comment by chenriksson** [comment]
Could we have a unit test that verifies that controller actions w/ HttpPost also have ValidateAntiForgeryToken?

**Comment by scottbommarito** [comment]
@skofman1 I tested all of the endpoints manually and checked that they were all hit.

@chenriksson I like that idea, will add to this PR.

**Comment by scottbommarito** [comment]
@joelverhagen I moved the code to a shared place by adding a `ViewHelpers` method and a shared Javascript function.

**Comment by scottbommarito** [comment]
@chenriksson Finished the test as requested and found two more endpoints with the issue!

File: tests/NuGetGallery.Facts/Controllers/ControllerTests.cs [hunk 2]
```diff
@@ -0,0 +1,104 @@
+﻿using System;
```
Comment (chenriksson) [inline tests/NuGetGallery.Facts/Controllers/ControllerTests.cs]: copyright

**Pull request merged**